workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl

```diff
@@ -218,7 +218,7 @@ steps:
           - arriba_fusion_step/discarded_fusions
           - arriba_drawing_step/output_pdf
       output_directory_name:
-        valueFrom: "arriba_outputs"
+        valueFrom: "arriba"
```

Comment:
Is it worth having this as an optional input with a default as 'arriba' instead?

Reply:
Where are you suggesting to create this optional input?

Reply:
As an input into the workflow, such as `output_directory_name_arriba`

Reply:
Updated in https://github.com/umccr/cwl-ica/commit/320a9132af1d70975d080276207baca098e23850